scripts/dokka-validate.sh

```diff
@@ -2,22 +2,23 @@
 
 EXIT_TYPE="exit 1"
 MODULE=""
+GRADLE_TASK="dokkaHtmlMultiModule"
 
-SEARCH_REGEX='No documentation for com.mapbox.navigation'
+SEARCH_REGEX='Undocumented: com.mapbox.navigation.'
 SEARCH_REGEX_EXCEPT='\.Companion ' # skip `Companion object` headers
 
 while getopts 'sm:' c
 do
   case $c in
     s) EXIT_TYPE="" ;; # soft failing (optional)
-    m) MODULE="${OPTARG}:";; # module to validate dokka (optional)
+    m) MODULE="${OPTARG}:" && GRADLE_TASK="${MODULE}:dokkaHtml";; # module to validate dokka (optional)
   esac
 done
 
 
 echo "Dokka validation is starting; clean project"
 ./gradlew clean
 echo "Dokka is validating ${MODULE}"
-./gradlew "${MODULE}dokka" | grep -i "$SEARCH_REGEX" | grep -v "$SEARCH_REGEX_EXCEPT" && { echo 'kdoc validation failed'; ${EXIT_TYPE}; }
+./gradlew $GRADLE_TASK | grep -i "$SEARCH_REGEX" #| grep -v "$SEARCH_REGEX_EXCEPT" && { echo 'kdoc validation failed'; ${EXIT_TYPE}; }
```

Comment:
Since we are commenting out the `exit 1` call, this script will never fail, isn't that right?

Reply:
I was testing new approach. revert it 👍